File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ChineseHolidayCalendarSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ChineseHolidayCalendarSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ChineseHolidayCalendar SDK exists test

local sdk = require("chinese-holiday-calendar_sdk")

describe("ChineseHolidayCalendarSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
